base/src/main/java/org/aya/distill/BaseDistiller.java

```diff
@@ -124,6 +124,7 @@ private <T extends AyaDocile> Doc visitArg(@NotNull Fmt<T> fmt, @NotNull Arg<T>
   }
 
   public static @NotNull Doc checkParen(@NotNull Outer outer, @NotNull Doc binApp, @NotNull Outer binOp) {
+    if (outer == binOp && (outer == Outer.IMin || outer == Outer.IMax)) return binApp;
```

Comment:
There's a way to avoid this

Reply:
Oh please let me know!
I admit my solution is a bit hacky.